### ISSUE TITLE
Set sensible SMTP timeout 

### ISSUE THREAD

**Issue by Kidswiss** [dani-garcia/bitwarden_rs#706]
When SMTP is enabled it will send a "new device" mail to the user's configured mail address. If that mail server is unreachable for some reason, or misconfigured, it will hang indefinitely with following log entries on the server:
```
[2019-10-27 17:43:33][rocket::rocket][INFO] POST /api/accounts/prelogin application/json; charset=utf-8:
[2019-10-27 17:43:33][_][INFO] Matched: POST /api/accounts/prelogin (prelogin)
[2019-10-27 17:43:33][_][INFO] Outcome: Success
[2019-10-27 17:43:33][_][INFO] Response succeeded.
``` 

On the client (Chrome):
![image](https://user-images.githubusercontent.com/7460550/68217328-b40db600-ffe2-11e9-8f84-13ec4fc87437.png)

Enabling debugging I saw that the server is hanging trying to connect to the SMTP host to send the "new device" mail, seemingly forever.

As discussed in #687 with @BlackDex there was already a discussion about this on the matrix chat on October 18.

**Comment by Kidswiss** [comment]
You're right after some time it prints something about login success in the log (sorry currently at work can't get you the exact logs...).

But that may be too long as the client will never receive that information and gets stuck.

**Comment by dani-garcia** [comment]
Resolved in #711, you can use SMTP_TIMEOUT=xx to set the timeout in seconds.

**Comment by Kidswiss** [comment]
Awesome! Thanks for the quick fix!
